db/migrate/20250816050817_add_metadata_action_to_ticket_logs.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddMetadataActionToTicketLogs < ActiveRecord::Migration[7.2]
+  def change
+    add_column :ticket_logs, :metadata_action, :string
```

Comment:
Please use `after: :action_type` (or pick another column that seems more reasonable to you, but don't just throw it to the end of the table randomly)

Reply:
Done